.circleci/config.yml

```diff
@@ -13,14 +13,15 @@
 # If you change the cache key prefix, also sync the restore_cache fallback to match.
 # Keep the static part of the cache key as prefix to enable correct fallbacks.
 # See https://circleci.com/docs/2.0/caching/#restoring-cache for how prefixes work in CircleCI.
-var_1: &docker_image circleci/node:10.12
-var_2: &cache_key angular_devkit-0.10.0-{{ checksum "yarn.lock" }}
+var_1: &default_docker_image circleci/node:10.12
+var_2: &browsers_docker_image circleci/node:10.12-browsers
```

Comment:
Is this needed? We use `puppeteer` for our e2e tests so I didn't expect it to be necessary.

Reply:
sadly headless chrome still requires dynamic linked dependency on libx11 which is pretty lame. We should raise with the Chrome team.